src/main.py

```diff
@@ -45,8 +96,8 @@ def rc_auto_loop_function_controller_1():
             # calculate the drivetrain motor velocities from the controller joystick axies
             # left = axis3 + axis1
             # right = axis3 - axis1
-            drivetrain_left_side_speed = controller_1.axis3.position()
-            drivetrain_right_side_speed = controller_1.axis2.position()
+            drivetrain_left_side_speed = controller.axis3.position()

```

Comment:
If you rename from controller_1 to controller, I would also recommend renaming all variables that contain controller_1